decidim-participatory_processes/app/jobs/decidim/participatory_processes/change_active_step_job.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
+    # rubocop:disable Rails/ApplicationJob
+    class ChangeActiveStepJob < ActiveJob::Base
```

Comment:
This job actually is not being used. @andreslucena do you want we to remove it?

Reply:
@tramuntanal I was actually thinking could we use the job anyways even when there's not much benefit from using it?

But this would be in line with the other scheduled rake tasks, such as:
https://github.com/decidim/decidim/blob/4964414d77d5b0bb61967b04a04b83b75d8b19f1/decidim-core/lib/tasks/decidim_metrics_tasks.rake#L14